resources/windows/windows.pkr.hcl

```diff
@@ -16,11 +25,23 @@ variable "region" {
 variable "install_user" {
   type    = string
   default = "vagrant"
+  description = <<EOF
+The name of a temporarily created Administrator account used for provisioning
```

Comment:
Comment is somewhat inaccurate, this is the admin account you will ssh into the box with later so not temporary.